packages/slice-machine/src/modules/environment/index.ts

```diff
@@ -141,9 +130,7 @@ export const environmentReducer: Reducer<
     case getType(refreshStateCreator):
       return {
         ...state,
-        env: action.payload.env,
-        warnings: action.payload.warnings,
-        configErrors: action.payload.configErrors,
+        ...action.payload.env,
```

Comment:
are you sure we still send an object with an env property and not the end directly in the payload ?

Reply:
Yes I'm sure look a the the refresh state action :),